app/views/timer_sessions/_timer_container.html.slim

```diff
@@ -1,4 +1,7 @@
-.box[data-controller="form timer"]
+.box[data-controller="form timer",
+  data-timer-timezone-value="#{Time.zone.now.
+  in_time_zone(@current_user.preference.time_zone).utc_offset / 1.hour}"]
```

Comment:
this to me seems too much logic to stay in the view untested. I suggest to move it into an helper and add a test for it